app/controllers/project_database.py

```diff
@@ -196,6 +196,42 @@ def get(self, project_id, database_id):
         return dict(status='success', data=dict(database=json.loads(database_data))), 200
 
 
+class ProjectDatabaseStatusView(Resource):
+
+    @jwt_required
+    def get(self, project_id, database_id):
+        """
+        """
+
+        project = Project.get_by_id(project_id)
+        if not project:
+            return dict(status='fail', message=f'Project with id {project_id} not found'), 404
+
+        database_existant = ProjectDatabase.get_by_id(database_id)
+
+        if not database_existant:
+            return dict(
+                status="fail",
+                message=f"Database with id {database_id} not found."
+            ), 404
+
+        # Check the database status on host
+        database_service = DatabaseService()
+        database_connection = database_service.create_db_connection(user=database_existant.user, password=database_existant.password, db_name=database_existant.name)
```

Comment:
Hey @MutegekiHenry You will need to close this connection after checking its success

Reply:
Maybe you can create a function that will close a particular connection

Reply:
sure thing!